lib/cors-anywhere.js

```diff
@@ -293,6 +294,10 @@ var getHandler = exports.getHandler = function(options, proxy) {
       delete req.headers[header];
     });
 
+    for (var header in corsAnywhere.addHeaders) {
```

Comment:
Because of the performance benefit? I've done some research, and it seems that a simple for loop is even faster: https://jsperf.com/for-vs-foreach/75